docs/content/direct/combined-status.md

```diff
@@ -8,38 +8,85 @@ It may make sense to expand the latter into a general feature in the future, acc
 
 The general technique involves the following ideas.
 
-1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a small subset of the expression language.
+1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a different expression language (one of growing currency in the Kubernetes milieu).
 
-1. The specification of how to combine reported state is defined in `StatusCollector` objects. These objects are referenced in the BindingPolicies right next to the criteria for selecting workload objects. This saves users the trouble of having to write selection criteria twice. With the specification being separate, it is possible to have a library of `StatusCollectors` that can be reused across different BindingPolicies. In the future KubeStellar would provide a library of `StatusCollector` objects that cover convenient use-cases for kubernetes built-in resources such as deployments.
+1. The expressions may reference the content of the workload object as it sits in the WDS as well as the state returned from the WECs.
 
-1. The combined reported state appears in a new kind of object, one per (workload object, BindingPolicy object) pair.
+1. The specification of how to combine reported state is defined in `StatusCollector` objects. These objects are referenced in the `BindingPolicy` objects right next to the criteria for selecting workload objects. This saves users the trouble of having to write selection criteria twice. With the specification being separate, it is possible to have a library of `StatusCollectors` that can be reused across different `BindingPolicy` objects. In the future KubeStellar would provide a library of `StatusCollector` objects that cover convenient use-cases for kubernetes built-in resources such as deployments.
+
+1. The `Binding` objects also hold references to `StatusCollector` objects. Each reference to a workload object is paired with references to all the `StatusCollectors` mentioned in all the `DownsyncObjectTestAndStatusCollection` structs that matched the workload object.
```

Comment:
I think the main point of the doc is to explain the "why"  (e.g., usage possibilities) , I'm not sure there is a point in just listing/specifying what are the object's fields

Reply:
This is the whole user-facing document about this feature, so it needs to explain not only the "why" but also lay out the relevant concepts and how they fit together. The paragraph that is comment is attached to is not a list of fields of an object but is talking about how the content of a Binding is related to the content of a BindingPolicy. In particular, this paragraph is pointing out that multiple DownsyncObjectTestAndStatusCollection structs in a BindingPolicy can match the same workload object and in this case their StatusCollector references are combined into one set of StatusCollector references in the corresponding position in the Binding object.